frontend/src/views/Share.vue

```diff
@@ -193,8 +212,14 @@ export default {
       this.error = null
 
       try {
-        let file = await api.getHash(encodeURIComponent(this.$route.params.pathMatch))
+        const password = this.password || ''
+        if (password !== ''){
+          this.attemptedPasswordLogin = true
+        }
+        let file = await api.getHash(encodeURIComponent(this.$route.params.pathMatch), password)
         this.path = file.path
+        this.token = file.token || ''
+        store.commit('setToken', this.token)
```

Comment:
It is, it is used in the JS that generates the downloadlink when you select a file and click download in the top bar

Reply:
I see, lets have it set on [/store/index.js](https://github.com/filebrowser/filebrowser/blob/master/frontend/src/store/index.js). Also use `this.$store.commit()` instead of importing the store.

Reply:
@ramiresviana fixed